File: ruby_squares/get_squares.js
var getSquares = function (arr) {
  var results = []
  arr.forEach(function (num) {
    results = results.push(Math.sqrt(num))
  })
  console.log(results)
}

getSquares([4, 1, 16, 1, 10, 35, 22])

File: ruby_squares/get_squares.rb
require 'pry'

def get_squares(arr)
  results = []
  arr.each do |num|
    # if Math.sqrt(num) % 1 == 0
    if (Math.sqrt(num) % 1).zero?
      results.push(num)
    end
  end
  puts results.sort.uniq
end
get_squares([4, 1, 16, 1, 10, 35, 22])

binding.pry
